### ISSUE TITLE
Directly exporting "unreachable" sub-collections stopped working

### ISSUE THREAD

**Issue by m417z** [jloosli/node-firestore-import-export#73]
The issue is relevant to my previous issue here: https://github.com/jloosli/node-firestore-import-export/issues/42

I previously reported that if I have a user whose document doesn't exist, but there is a sub-collection with documents, the sub-collection of this user is not exported.

You said that it wasn't possible to fix, so I wrote a script to do manual export per "unreachable" sub-collection, similar to:
```
const collectionRef = admin.firestore().collection('users/' + userId + '/semesters');
firestoreImportExport.firestoreExport(collectionRef).then(/*...*/);
```

It worked at least until a month ago. But today I discovered that it stopped working. And it seems that it stopped working only for such "unreachable" sub-collections, i.e. sub-collections that neither get exported via the full database export.

I understand that it's a regression on the Firestore end, but perhaps you have a workaround for this. If not, it would be great if you could send the Firestore team a report about this, or alternatively point me to the regressed API call and I'll try to contact them.

As of now, I don't have a way to do a full backup of my project's database, which kinda sucks :(

**Comment by jloosli** [comment]
My suggestion would be to backup the data _before_ deleting the parent. This is a bug/loophole that the Firebase team is actively working to close, so I wouldn't rely on an undocumented process that is likely to go away completely sooner rather than later.

**Comment by m417z** [comment]
> backup the data _before_ deleting the parent

I'm not deleting the parent. I've never had it in the first place, I've created the sub-collections right away without ever having the parent document.

> undocumented process

What are you referring to as an undocumented process? I don't think that my system does anything undocumented.

In any case, I tried other export tools in the hope to find one that works, and found out that:
* Non of the three or so available tools support exporting sub-collections correctly.
* There are a number of reported issues for these projects related to sub-collections, meaning that there are users who need this feature.

Fortunately, I found this pull request of @kitfit-dave who came up with a way to export sub-collections correctly:
https://github.com/willhlaw/node-firestore-backup-restore/pull/55

I've incorporated a similar change to your project and it fixes the issue. I'll create a pull request soon.
